broadcast-monitoring/infrastructure/elemental/custom_resources/custom-resource-py/libs/mediapackage.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
```

Comment:
Do we need ASL for this if repository is MIT-0?

Reply:
ASL already exists in a number of files... leaving this as is.